.github/workflows/build.yml.in

```
@@ -58,6 +58,17 @@ jobs:
         run: |
           lake exe lint-style
 
+      - name: "check the formatting of the PR title and for missing or inconsistent labels"
+        # TODO: skip this check if the PR is in draft stage!
+        shell: bash # just in case
+        run: |
+          set -eo pipefail
+          title="${{github.event.pull_request.title}}"
+          echo $title
```

This needs to be quoted otherwise `*` etc in the title might be interpreted as a glob
```suggestion
          echo "$title"
```